doc/articles/features/working-with-themes.md

```diff
@@ -2,30 +2,114 @@
+
+Below is an example of how to use these classes to change the app theme at runtime based on a raised event:
+
+```csharp
+using CommunityToolkit.WinUI.UI.Helpers;
+using Uno.Toolkit.UI;
+
+namespace MyApp.UI.Views;
+
+public class MainPage : Page
+{
+    public MainPage()
+    {
+        this.InitializeComponent();
+
+        ThemeListener.Current.ThemeChanged += OnThemeChanged;
+    }
+
+    private void OnThemeChanged(ThemeListener sender)
+    {
+        bool isDarkMode = sender.CurrentTheme == ApplicationTheme.Dark;
+
+        SystemThemeHelper.SetApplicationTheme(darkMode: isDarkMode);
+    }
+}
 ```
-Uno.UI.ApplicationHelper.RequestedCustomTheme = "HighContrast";
-```
 
-This must be called during app startup in the `App` class constructor. Uno does not yet support `FrameworkElement.RequestedTheme` so the theme cannot be changed dynamically at runtime.
+## Change the app theme at runtime
+
+The `SetApplicationTheme` method in this helper class can also be used to support an in-app toggle for dark mode. For example, you could add a toggle button to your app's settings page that allows the user to switch between light and dark modes. The following code snippet shows how to implement this:
+
+```csharp
+using CommunityToolkit.WinUI.UI.Helpers;
+using Uno.Toolkit.UI;
 
-Setting `Application.Current.RequestedTheme` outside of `App` constructor is not allowed, which is in line with UWP.
+namespace MyApp.UI.Views;
 
-## Reacting to OS theme changes
+public class SettingsPage : Page
+{
+    public SettingsPage()
+    {
+        this.InitializeComponent();
 
-If you don't specify the theme in `App` constructor and `AppResources.xaml` or `App.xaml`, your app will automatically adapt to OS theme changes at runtime. However, on Android, you need to make sure to add `ConfigChanges.UiMode` to the `MainActivity` `[Application]` attribute, for example:
+        this.Loaded += OnLoaded;
+    }
 
-``` c#
-[Activity(
-        MainLauncher = true,
-        ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize | ConfigChanges.UiMode,
-        WindowSoftInputMode = SoftInput.AdjustPan | SoftInput.StateHidden
-    )]
+    private void OnLoaded(object sender, RoutedEventArgs e)
+    {
+        this.DarkModeToggle.IsOn = SystemThemeHelper.IsDarkModeEnabled;
+    }
+
+    private void OnDarkModeToggleToggled(object sender, RoutedEventArgs e)
+    {
+        SystemThemeHelper.SetApplicationTheme(darkMode: this.DarkModeToggle.IsOn);
+    }
+}
 ```
+
+## Change the app theme at startup
+
+Another method to change the app theme is to adjust it immediately upon startup. The following must only be called in the `App` class constructor.
+
+```csharp
+Uno.UI.ApplicationHelper.RequestedCustomTheme = "Dark";
```

Comment:
you should just include the full code (containing method signature, uno vs windows code), so it is clear at a glance:
```suggestion
public App()
{
	this.InitializeComponent();

#if HAS_UNO
	Uno.UI.ApplicationHelper.RequestedCustomTheme = nameof(ApplicationTheme.Dark);
#else
	this.RequestedTheme = ApplicationTheme.Dark;
#endif
}
```